defaults/krill.conf

```diff
@@ -106,6 +106,102 @@
 #  <service_uri>...                       (various UI resources)
 ### service_uri = "https://localhost:3000/"
 
+
+######################################################################################
+#                                                                                    #
+#                                     METRICS                                        #
+#                                                                                    #
+######################################################################################
+
+# Krill exposes metrics for use with prometheus under: <service_uri>/metrics
+#
+# By default a lot of details are included in these metrics. Generating these metrics
+# is not particularly hard on Krill, but in case you have many CAs, children or publishers
+# under your Krill instance you may still want to disable certain metrics to avoid
+# overwhelming your prometheus instance with too much data.
+#
+# Note that you can also get all of this information through the API, so there
+# are other ways than using Prometheus to monitor details.
+
+# Always enabled:
+#################
+#
+# krill_server_start              unix timestamp in seconds of last krill server start
+# krill_version_major             krill server major version number
+# krill_version_minor             krill server minor version number
+# krill_version_patch             krill server patch version number
+# krill_auth_session_cache_size   total number of cached login session tokens
```

Comment:
It's not strictly true that this is always enabled as it is behind a feature flag, I can guess it's simpler to stick to the default feature set for these docs otherwise they will become complicated for the majority case for the sake of the few if any customers who change the Rust feature set used to build Krill.

Reply:
true, fixed